docs/sources/auth/okta.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "Okta OAuth2 authentication"
+description = "Grafana Okta OAuth Guide "
+keywords = ["grafana", "configuration", "documentation", "oauth"]
+type = "docs"
+[menu.docs]
+name = "Okta"
+identifier = "okta_oauth2"
+parent = "authentication"
+weight = 3
++++
+
+# Okta OAuth2 authentication
+
+> Only available in Grafana v7.0+
+
+The Okta authentication allows your Grafana users to log in by using an external Okta authorization server.
+
+## Create an Okta application
+
+Before you can sign a user in, you need to create an Okta application. Start by signing in to the Okta Developer Console: log in to the Okta, go to Admin and then select *Developer Console* in the top menu. Then create an application:
```

Comment:
Should this be "log in to Okta" or "log in to the Okta server"?

Reply:
It should be done via Okta portal (okta.com) (it's not a standalone server or so).